django/test/testcases.py

```diff
@@ -1221,14 +1215,12 @@ def setUpClass(cls):
                         call_command('loaddata', *cls.fixtures, **{'verbosity': 0, 'database': db_name})
                     except Exception:
                         cls._rollback_atomics(cls.cls_atomics)
-                        cls._remove_databases_failures()
```

Comment:
We don't need to handle this manually anymore because:

_"If `setUpClass()` fails, meaning that `tearDownClass()` is not called, then any cleanup functions added will still be called."_

(see [`TestCase.addClassCleanup()`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.addClassCleanup) docs).

Reply:
This is also covered by tests, see `test_utils.tests.TestBadSetUpTestData`.